addons/mail/controllers/discuss/rtc.py

```diff
@@ -139,11 +139,10 @@ def channel_ping(self, channel_id, rtc_session_id=None, check_rtc_session_ids=No
             ]
             channel_member_sudo.channel_id.rtc_session_ids.filtered_domain(domain).write({})  # update write_date
         current_rtc_sessions, outdated_rtc_sessions = channel_member_sudo._rtc_sync_sessions(check_rtc_session_ids)
-        return (
-            Store(member.channel_id, {"rtc_session_ids": Store.Many(current_rtc_sessions, mode="ADD")})
-            .add(
```

Comment:
Note: recent fix finally made multiple-values for the same key supported in field list